Controller/Payment/Order.php

```diff
@@ -225,7 +225,7 @@ public function execute()
             $response->setData($responseContent);
             $response->setHttpResponseCode(400);
 
-            $this->logger->critical("Razorpay Order: Payment already made for order :" . $receipt_id,);
+            $this->logger->critical("Razorpay Order: Payment already made for order :" . $receipt_id);
```

Comment:
Please acknowledge the PSR-3 standard for `LoggerInterface`: https://www.php-fig.org/psr/psr-3/

You'll be able to find there's a 2nd argument in each logger method -- `$context`, you can pass an array of contextual information for an event.
```suggestion
            $this->logger->critical("Razorpay Order: Payment already made", ['order_id' => $receipt_id]);
```